dandi/publish/views/asset.py

```diff
@@ -31,6 +31,8 @@ class Meta:
 
 
 class AssetFilter(filters.FilterSet):
+    path = filters.CharFilter(lookup_expr='istartswith', field_name='path')
+
     class Meta:
         model = Asset
```

Comment:
You shouldn't need to specify `field_name` if it's the same as the filter name: https://django-filter.readthedocs.io/en/master/guide/usage.html#the-filter